rados/rados_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/ceph/go-ceph/rados"
+	"github.com/immesys/go-ceph/rados"
```

Comment:
I realize this is a WIP but this will need to be removed prior to merge. I recommend you test your code with the upstream library. If this fork has something that you depend on that is not present here in ceph/go-ceph please contribute the needed changes first, or file related issues. Thanks!